applications/RomApplication/python_scripts/rom_solver.py

```diff
@@ -51,8 +61,9 @@ def _ValidateAndReturnRomParameters(self):
                     err_msg = "\'nodal_unknowns\' in \'rom_settings\' is not provided and there is a not-valid implementation in base solver."
                     err_msg += " Please manually set \'nodal_unknowns\' in \'rom_settings\'."
                     raise Exception(err_msg)
+            solving_strategy = self.settings["solving_strategy"].GetString()
```

Comment:
This is what I meant above, shouldn't `solving_strategy` be encapsulated within `rom_settings`?